vividus-to-zephyr-exporter/src/test/java/org/vividus/zephyr/exporter/ZephyrExporterTests.java

```diff
@@ -126,10 +128,10 @@ private ZephyrConfiguration prepareTestConfiguration()
         return configuration;
     }
 
-    private void mockJiraIssueRetrieve(String issueKey, String issueId) throws IOException
+    private void mockJiraIssueRetrieve(String issueKey, String issueId) throws IOException, JiraConfigurationException
     {
         JiraEntity issue = new JiraEntity();
         issue.setId(issueId);
-        when(jiraFacade.getIssue(issueKey)).thenReturn(issue);
+        when(jiraFacade.getIssue(issueKey, Optional.empty())).thenReturn(issue);
```

Comment:
lets stop mocking 
```
@Mock
private ZephyrExporterProperties zephyrExporterProperties;
```

Reply:
It probably should be done in separate commit since I don't perform any stuff with the ZephyrExporterProperties here